File: rmt/src/test/scala/de/flower/rmt/ui/page/events/manager/EventsPageTest.scala
package de.flower.rmt.ui.page.events.manager

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class EventsPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new EventsPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/page/events/player/EventsPageTest.java
package de.flower.rmt.ui.page.events.player;

import de.flower.rmt.test.AbstractRMTWicketIntegrationTests;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class EventsPageTest extends AbstractRMTWicketIntegrationTests {

    @Test
    public void testRender() {
        testData.createEvent();
        wicketTester.startPage(new EventsPage());
        wicketTester.dumpPage();
    }
}